### ISSUE TITLE
WCAG 2.4.4 

### ISSUE THREAD

**Issue by kosarko** [ufal/dspace-angular#15]
[2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/WCAG21/#link-purpose-in-context)

Links do not have a discernible name: Failing Elements
a#repository_path.hidden, sereval buttons
